frontend/ui/quickstart.lua

```diff
@@ -0,0 +1,70 @@
+--[[--This module is responsible for generating the quickstart guide.
+]]
+local DataStorage = require("datastorage")
+local FileConverter = require("apps/filemanager/filemanagerconverter")
+local gettext = require("gettext")
+local lfs = require("libs/libkoreader-lfs")
+local T = require("ffi/util").template
+
+local quickstart_shown = G_reader_settings:readSetting("quickstart_shown") or nil
+local quickstart_shown_version = G_reader_settings:readSetting("quickstart_shown_version") or nil
+local quickstart_force_show_version = 201511982
+local language = G_reader_settings:readSetting("language") or "en"
+local version, _, rev = G_reader_settings:current_version()
+
+local quickstart_guide = T(gettext([[
+# KOReader Quickstart Guide
+
+Welcome to KOreader. You can activate the menu by swiping down from the top of the screen. Clicking outside the menu or swiping up on the menu will discard it. Turning pages can be done either by swiping left and right or by single taps on the left or right side of the screen.
+
+**Main menu**
+
+![Menu](../resources/icons/menu-icon.png) You can always view this quickstart guide again through *Help* → *Quickstart guide* in the top right menu.
+
+**Settings**
+
+![Settings](../resources/icons/appbar.settings.png) You can change the language and other settings through the gear icon.
+
+------------
+
+Generated by KOReader %1.
+]]),
+    rev)
```

Comment:
I suggest we also include the commit hash here in case people use it for bug report. It's easier for us to look up the version by hash.